### PR TITLE
Fix NaN SIIA for models that have all nodes in circuit

### PR DESCRIPTION
**Pull request by FlyingPumba**

### CODE REVIEW

File: iit/model_pairs/strict_iit_model_pair.py [hunk 1]
```diff
@@ -125,7 +125,12 @@ def run_eval_step(self, base_input, ablation_input, loss_fn: Callable[[Tensor, T
             else:
                 accuracy = ((ll_output - base_y).abs() < self.training_args["atol"]).float().mean().item()
             accuracies.append(accuracy)
-        accuracy = np.mean(accuracies)
+
+        if len(accuracies) > 0:
+            accuracy = np.mean(accuracies)
+        else:
+            accuracy = 100.0
```
Comment (cybershiptrooper) [inline iit/model_pairs/strict_iit_model_pair.py]: Isn't this handled while printing? (i.e., the logger expects accuracy to be in 0-1, so the first commit was correct? Did you get any logging mistakes on doing that?) 
LMK if this is not happening!

**Review by cybershiptrooper — CHANGES_REQUESTED** [review]

**Review by cybershiptrooper — APPROVED** [review]
